bfcfg

```diff
@@ -653,17 +653,22 @@ sys_cfg_one_byte()
 # Below are the offsets defined in UEFI which is 4(fixed header) plus the offset
 # of the variable within the BfSysCfg struct. These offsets are not supposed to
 # change in order to be backward compatible with previous releases.
-#   VARIABLE(Name)       OFFSET(Byte)  SIZE(Byte)
-#   SYS_ENABLE_SMMU      24            1
-#   SYS_DISABLE_SPMI     25            1
-#   SYS_ENABLE_2ND_EMMC  26            1
-#   SYS_BOOT_PROTECT     27            1
-#   SYS_ENABLE_SPCR      32            1
-#   SYS_DISABLE_PCIE     33            1
-#   SYS_ENABLE_OPTEE     34            1
-#   SYS_ENABLE_I2C0      36            1
-#   SYS_DISABLE_FORCE_PXE_RETRY  37    1
-#   SYS_ENABLE_BMC_FIELD_MODE    38    1
+#   VARIABLE(Name)               OFFSET(Byte)  SIZE(Byte)
+#   SYS_ENABLE_SMMU              24            1
+#   SYS_DISABLE_SPMI             25            1
+#   SYS_ENABLE_2ND_EMMC          26            1
+#   SYS_BOOT_PROTECT             27            1
+#   SYS_ENABLE_SPCR              32            1
+#   SYS_DISABLE_PCIE             33            1
+#   SYS_ENABLE_OPTEE             34            1
+#   SYS_ENABLE_I2C0              36            1
+#   SYS_DISABLE_FORCE_PXE_RETRY  37            1
+#   SYS_ENABLE_BMC_FIELD_MODE    38            1
+#   SYS_LARGE_ICMC_SIZE          39            4
+#   SYS_DEFAULT_LARGE_ICMC_LOCK  43            1
+#   SYS_DISPLAY_FLAGS            44            1
+#   SYS_CE_THRESHOLD             45            4
+#   SYS_ENABLE_I2C3              49            1
```

Comment:
Please rebase to incorporate the latest changes.

Reply:
@kblaiech not sure I understand - this is up to date already? Last commit to bfcfg was 2 months ago. We have a few conflicting PRs out now though and so we will have to resolve merge conflicts and incorporate changes as we merge each change. I'll update this PR if another one of those merges first (and Github should warn of any merge conflicts too)